### PR TITLE
Filling in some links, and making the IconLink component handle both Lin...

### PR DESCRIPTION
**Pull request by ScottDowne**
...k routes and simple hrefs

### CODE REVIEW

**Comment by ScottDowne** [comment]
Fixes #324

File: components/icon-link.jsx [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+        </Link>
+      )
+    }
+    return (
+      <a href={this.props.href}>
+        {this.props.children}
+      </a>
+    )
+  }
+});
```
Comment (vazquez) [inline components/icon-link.jsx]: This is great.

**Comment by vazquez** [comment]
R+ I really like the flexibility of this component.

The only comment I would have is regarding the component name. Would something like `LinkHandler` make more sense to describe this component? Anyway, just an opinion. Otherwise, awesome :star:

**Comment by ScottDowne** [comment]
@vazquez I think a LinkHandler would be a nice addition, but it would be a new thing separated out of IconLink which does a bit more than just create a link. I say file a new issue to create a LinkHandler which then boils down IconLinks a bit.

**Pull request merged**